### ISSUE TITLE
纵向逻辑回归，conf.json文件中“component_parameters”中，若不使用"common"来配置“hetero_lr_0”，则job无法停止

### ISSUE THREAD

**Issue by MDGBDGMG** [FederatedAI/FATE#3746]
**Describe the bug**
做纵向逻辑回归，conf.json文件中“component_parameters”中，若不使用"common"来配置“hetero_lr_0”，则job无法停止

**To Reproduce**
1.上传guest和host方数据
2.编写conf.json文件
如下：
{
    "dsl_version": 2,
    "initiator": {
        "role": "guest",
        "party_id": 9999
    },
    "role": {
        "arbiter": [
            10000
        ],
        "host": [
            10000
        ],
        "guest": [
            9999
        ]
    },
    "component_parameters": {
        "role": {
            "guest": {
                "0": {
                    "reader_0": {
                        "table": {
                            "name": "breast_hetero_guest",
                            "namespace": "experiment"
                        }
                    },
                    "data_transform_0": {
                        "with_label": true,
                        "label_type": "int"
                    },
                    "data_transform_1": {
                        "with_label": true,
                        "label_type": "int"
                    },
                    "reader_1": {
                        "table": {
                            "name": "breast_hetero_guest",
                            "namespace": "experiment"
                        }
                    },
                    "hetero_lr_0": {
                        "penalty": "L2",
                        "tol": 0.0001,
                        "alpha": 0.01,
                        "optimizer": "rmsprop",
                        "batch_size": -1,
                        "learning_rate": 0.15,
                        "init_param": {
                            "init_method": "zeros"
                        },
                        "max_iter": 30,
                        "early_stop": "diff",
                        "cv_param": {
                            "n_splits": 5,
                            "shuffle": false,
                            "random_seed": 103,
                            "need_cv": false
                        },
                        "validation_freqs": 1,
                        "early_stopping_rounds": 3,
                        "metrics": [],
                        "use_first_metric_only": false,
                        "sqn_param": {
                            "update_interval_L": 3,
                            "memory_M": 5,
                            "sample_size": 5000,
                            "random_seed": null
                        }
                    },
                    "evaluation_0": {
                        "eval_type": "binary"
                    }
                }
            },
            "host": {
                "0": {
                    "reader_0": {
                        "table": {
                            "name": "breast_hetero_guest",
                            "namespace": "experiment"
                        }
                    },
                    "data_transform_0": {
                        "with_label": false
                    },
                    "reader_1": {
                        "table": {
                            "name": "breast_hetero_guest",
                            "namespace": "experiment"
                        }
                    },
                    "hetero_lr_0": {
                        "penalty": "L2",
                        "tol": 0.0001,
                        "alpha": 0.01,
                        "optimizer": "rmsprop",
                        "batch_size": -1,
                        "learning_rate": 0.15,
                        "init_param": {
                            "init_method": "zeros"
                        },
                        "max_iter": 30,
                        "early_stop": "diff",
                        "cv_param": {
                            "n_splits": 5,
                            "shuffle": false,
                            "random_seed": 103,
                            "need_cv": false
                        },
                        "validation_freqs": 1,
                        "early_stopping_rounds": 3,
                        "metrics": [],
                        "use_first_metric_only": false,
                        "sqn_param": {
                            "update_interval_L": 3,
                            "memory_M": 5,
                            "sample_size": 5000,
                            "random_seed": null
                        }
                    },
                    "evaluation_0": {
                        "eval_type": "binary"
                    }
                }
            }
        }
    }
}
3.提交执行

**Expected behavior**
期望job正常执行，得到结果

**Screenshots**
实际执行60多个小时，但是hetero_lr_0模块的日志显示 “pid: 10663, elapsed: 83808 ms”  ，最后我手动停止了这个任务

![image](https://user-images.githubusercontent.com/45615979/155922595-6148c4cc-445e-47a4-a423-3ca89615ef4c.png)

![image](https://user-images.githubusercontent.com/45615979/155922464-6cdf5412-de85-469a-8b2a-08e7628bc364.png)

**Desktop (please complete the following information):**
 - OS: linux
 - Browser chrome
 - Version  fate1.7.0  fate1.7.1

**Comment by MDGBDGMG** [comment]
我将hetero_lr_0的参数放在role的guest和host中，结果就是job无法停止。
若我将hetero_lr_0的参数放在common中，common与role同级，那么job可以正常结束。
![image](https://user-images.githubusercontent.com/45615979/155930847-78957356-e3d1-4c26-8068-66b3e79c7abb.png)

**Comment by gxcuit** [comment]
> 我将hetero_lr_0的参数放在role的guest和host中，结果就是job无法停止。 若我将hetero_lr_0的参数放在common中，common与role同级，那么job可以正常结束。 ![image](https://user-images.githubusercontent.com/45615979/155930847-78957356-e3d1-4c26-8068-66b3e79c7abb.png)

我感觉问题可能出在，除了host和guest，你还需指定arbiter 的参数

**Comment by MDGBDGMG** [comment]
搞定了，多谢
